sys/vane/ford.hoon

```diff
@@ -1157,30 +1216,44 @@
           ?.  (slob for p.gab)  ~
           `(slap gab [%limb for])
       ?~  zat
-        :: ~$  link-miss
-        (flaw cof [%leaf "ford: no link: {<[for too]>}"]~)
-      :: ~$  link-grab
-      ~|  [%link-maul for too]
+        :: ~$  translate-mark-miss
+        (flaw cof [%leaf "ford: no translate-mark: {<[for too]>}"]~)
+      :: ~$  translate-mark-grab
+      ~|  [%translate-mark-maul for too]
       (maul cof u.zat vax)
     ::
-    ++  lion                                            ::  translation search
-      ~/  %lion
+    ++  translation-targets
```

Comment:
Could you explain what's different about this arm? I can't tell how many of the changes are cosmetic and how many are related to functionality.

Reply:
All your commits until **clean-ford** @ 56c1ca2 are cleanup, the commits after are the actual change. Should I open that as a separate PR?

Reply:
(Current) changes just from non- **greedy-ford**: https://github.com/urbit/arvo/pull/449/files/9f74a8d..ce120f8